src/Table/Table.stories.jsx

```diff
@@ -77,4 +88,24 @@ storiesOf('Table', module)
       caption="Famous Internet Cats"
       className={['table-responsive']}
     />
-  ));
+  ))
+  .add('sortable', () => {
+    const catDataSortable = catData.slice();
```

Comment:
I'm making a copy to keep the other stories unaffected by the sorting in this story.